scripts/data/gen-nodetime/src/relayer/lib/relayer.ts

```diff
@@ -132,8 +118,7 @@ export default class Relayer {
             account.address,
             {
                 prefix: chain.address_prefix,
-                gasPrice: chainGP,
-                gasLimits: calcGasLimits(chain.gas_limit)
+                gasPrice: chainGP
```

Comment:
@jeronimoalbi can you explain me why `gasLimits` is no longer required ? thx in advance :)

Reply:
@tbruyelle of course, yes 👍 

The TS relayer we use removed the `gas_limits` from the `Chain` type in favour of [auto gas calculation](https://github.com/confio/ts-relayer/pull/215)

Reply:
thank you :pray: !